### DISCUSSION TITLE
Running vanilla extract on the server

### DISCUSSION THREAD

**Discussion by atabel** [vanilla-extract-css/vanilla-extract#891] [Q&A]
Hi, I haven't found any reference in docs about running vanilla extract on the server (apart from Next).
I've tried to setup a simple node (express) server that outputs an html generated with `ReactDomServer.renderToString` but I get this error from vanilla extract: 

```
Error: Styles were unable to be assigned to a file. This is generally caused by one of the following:
    
    - You may have created styles outside of a '.css.ts' context
    - You may have incorrect configuration. See https://vanilla-extract.style/documentation/getting-started
```

Do I need to compile my server code with any bundler before running it? I'm just running it with `ts-node`

Thanks in advance for the help ;)

**Comment by wmertens** [comment]
No idea, but there's some issues with the same error, and maybe you can check that a regular .css file works?
